### PR TITLE
Add GoodDollar DAO Integration

### PR DESCRIPTION
**Pull request by sirpy**

### CODE REVIEW

**Review by Rashmi-278 — APPROVED** [review]

**Comment by sirpy** [comment]
@Rashmi-278 how can we progress with the integration

File: protocols/gooddollar/index.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "cname": "gooddollar",
+  "description": "GoodDollar - Universal basic income protocol, for the people by the people",
+  "path": "gooddollar",
+  "isEnabled": true,
+  "type": "compoundish",
+  "discourseForum.url": "https://discourse.gooddollar.org",
+  "discourseForum.categoryId": "5",
+  "safeAddress": "0x5C16960F2Eeba27b7de4F1F6e84E616C1977e070"
```
Comment (Rashmi-278) [inline protocols/gooddollar/index.json:9]: Hey, can you change this to treasuryAddress: "0x5C16960F2Eeba27b7de4F1F6e84E616C1977e070"?

Comment (sirpy) [reply protocols/gooddollar/index.json]: @Rashmi-278 done

Comment (Rashmi-278) [reply protocols/gooddollar/index.json]: Hey, sorry there was a typo on my end, can you please change it to 

`  "treasuryAddresses": ["0x5C16960F2Eeba27b7de4F1F6e84E616C1977e070"]
`

Comment (Rashmi-278) [reply protocols/gooddollar/index.json]: Also can you submit this form if you haven't done so ? https://docs.boardroom.info/adding-your-project/adding-your-project/unlocking-the-hub

Comment (sirpy) [reply protocols/gooddollar/index.json]: done

**Review by Rashmi-278 — APPROVED** [review]

**Pull request merged**